packages/components/src/custom-select-control/test/index.js

```diff
@@ -9,24 +9,25 @@ import userEvent from '@testing-library/user-event';
  */
 import CustomSelectControl from '..';
 
+const options = [
+	{
+		key: 'violets',
+		name: 'violets',
```

Comment:
I'd recommend a different `key` and `name` for every pair. Otherwise, we won't catch the case when we're incorrectly using one instead of the other in the component code.